.github/workflows/coding-standard-baseline.yml

```diff
@@ -0,0 +1,67 @@
+name: M2 Coding Testing
+
+on:
+  push:
+    branches: [ 2.4-develop, develop, codingstandard_baseline_integration ]
+  pull_request:
+    branches: [ 2.4-develop, develop, codingstandard_baseline_integration ]
+  workflow_call:
```

Comment:
We do not need this section, do we?

Reply:
We require only for pull_request on 2.4-develop , removing others now